domain/src/main/java/me/androidbox/domain/authentication/remote/Authentication.kt

```diff
@@ -0,0 +1,8 @@
+package me.androidbox.domain.authentication.remote
+
+import me.androidbox.domain.authentication.model.LoginModel
+
+interface Authentication {
```

Comment:
If this is the repository, I'd name it AuthenticationRepository

Reply:
@MKiperszmid @philipplackner 
Same as above, this is requesting and fetching from the API endpoint